lib/src/main/java/com/auth0/msg/JsonResponseDescriptor.java

```diff
@@ -0,0 +1,34 @@
+package com.auth0.msg;
+
+import java.util.List;
+import java.util.Map;
+
+public class JsonResponseDescriptor extends AbstractMessage{
+
+    public JsonResponseDescriptor() {
+    }
+
+    public JsonResponseDescriptor(Map<String, Object> claims) {
+        super(claims);
+    }
+
+    @Override
+    protected List<String> getRequiredClaims() {
+        return null;
+    }
+
+    @Override
+    public Map<String, Object> getClaims() throws InvalidClaimsException {
+        return super.getClaims();
+    }
+
+    @Override
+    public MessageType getMessageType() {
+        return MessageType.JSON_RESPONSE_DESCRIPTOR;
+    }
+
+    @Override
+    public boolean hasError() {
+        return false;
```

Comment:
why are you overriding hasError and always returning false? What happens when verify fails? If this is a special scenario please provide javadoc to explain that.

Reply:
Corrected